### PR TITLE
9.0: remove support of DOTNET_SSL_DIRS in favor of SSL_CERT_DIR.

### PR DESCRIPTION
**Pull request by tmds**
Fixes https://github.com/redhat-developer/s2i-dotnetcore/issues/491.

@omajid @aslicerh ptal.

### CODE REVIEW

File: 9.0/build/test/run [hunk 1]
```diff
@@ -297,8 +297,8 @@ test_config_1() {
   # DOTNET_STARTUP_PROJECT=src/app
   assert_equal "${response}" "Hello world"
   # DOTNET_TEST_PROJECTS=test/test1 test/test2
-  assert_contains "${s2i_build}" "Finished:    test1"
-  assert_contains "${s2i_build}" "Finished:    test2"
+  assert_contains "${s2i_build}" "Passed!.*test1"
+  assert_contains "${s2i_build}" "Passed!.*test2"
```
Comment (tmds) [inline 9.0/build/test/run:301]: This change is unrelated to the certificate environment variables. The output of the .NET CLI changed, and this matches with the new output.

**Review by omajid — APPROVED** [review]
Thanks!

File: 9.0/runtime/contrib/etc/trust_ssl_dirs [hunk 2]
```diff
@@ -1,54 +1,5 @@
-# Trust certificates from all DOTNET_SSL_DIRS.
 if [ -n "$DOTNET_SSL_DIRS" ]; then
 
-  echo "warn: DOTNET_SSL_DIRS is deprecated. Please use SSL_CERT_DIR instead."
+  echo "error: DOTNET_SSL_DIRS is no longer supported. Certificates can be trusted using SSL_CERT_DIR."
```
[the remark this answers is not in the document]
Comment (tmds) [reply 9.0/runtime/contrib/etc/trust_ssl_dirs:3]: Yes, I considered it and decided to leave it at printing the error message only.

**Comment by tmds** [comment]
Thanks for the review, @omajid!
